CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - Remove \" in value handled by getGeometry
+- [cygnus-common][cygnus-ngsi] Add following parameters (and environment variables for docker) to be able to connect to sql database with SSL/TLS: "mysql_options" (`CYGNUS_MYSQL_OPTIONS`), "postgresql_options" (`CYGNUS_POSTGRESQL_OPTIONS`) and "postgis_options" (`CYGNUS_POSTGIS_OPTIONS`) (#1867)
```

Comment:
Thank you for your review. I add a row to `CHANGES_NEXT_RELEASE`.

Reply:
Ok. Perfect. Thanks!

NTC